### PR TITLE
Improve WebPubSub README.

### PR DESCRIPTION
**Pull request by JialinXin**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by vicancy — APPROVED** [review]

File: sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/README.md [hunk 1]
```diff
@@ -33,58 +33,49 @@ dotnet add package Microsoft.Azure.WebPubSub.AspNetCore --prerelease
 
 ### Authenticate the client
 
-In order to interact with the service, you'll need to create an instance of the WebPubSubServiceClient class. To make this possible, you'll need the connection string or a key, which you can access in the Azure portal.
+In order to interact with the service, you'll need to inject the Web PubSub service with valid credential. To make this possible, you'll need the connection string or a key, which you can access in the Azure portal. Besides, if you want to invoke service REST API, you can call `AddWebPubSubServiceClient<THub>()` where `THub` is user implemented [`WebPubSubHub`](#webpubsubhub) listening to important events.
```
Comment (jsquire) [inline sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/README.md:36]:
```suggestion
In order to interact with the service, you'll need to provide the Web PubSub service with a valid credential. To make this possible, you'll need the connection string or a key, which you can access in the Azure portal. Besides, if you want to invoke service REST API, you can call `AddWebPubSubServiceClient<THub>()` where `THub` is user implemented [`WebPubSubHub`](#webpubsubhub) listening to important events.
```

File: sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/README.md [hunk 2]
```diff
@@ -33,58 +33,49 @@ dotnet add package Microsoft.Azure.WebPubSub.AspNetCore --prerelease
...
 
-### Create a `WebPubSubServiceClient`
+### Inject Web PubSub service with options
```
Comment (jsquire) [inline sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/README.md:38]:
```suggestion
### Configure Web PubSub options
```

**Comment by annelo-msft** [comment]
This PR broke the build: https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/1410034/logs/183

We need to update snippet files and not just the READMEs.

**Comment by jsquire** [comment]
Apologies.  As the person who cleared check-enforcer, this is on me.    I've put out #27350 to resolve this.

**Comment by annelo-msft** [comment]
I have https://github.com/Azure/azure-sdk-for-net/pull/27360 about to merge.

**Pull request merged**